backend/README.md

```diff
@@ -630,4 +630,12 @@ Implement in this order:
 
 ## 📄 License
 
-This project is licensed under the MIT License. 
\ No newline at end of file
+This project is licensed under the MIT License. 
+
+### Tasks
+
+[x] Configure docker compose to run postgres via docker locally
+[x] configure database connection -> appln prop.
+[x] Flyway migration -> (tables to start) -- study normalisation concept for tables creation
+[x] migrate users tables id, email, name, passwd
+[ ] endpoint - users, auth (use Bcrypt - hash for passwords)
```

Comment:
For next points, I would suggest:
- Implement the endpoints to create a new user (accepts email and password) saves password with hash in the database.
- Implement endpoint to authenticate user
   - take email and password
   - If matches, you sign a JWT token and return in the body
- Implement endpoint to return user profile
   - Takes JWT token in authorization header (Check Spring Security module, filters (OncePerRequestFilter), SecurityConfig using HttpSecurit, AuthenticationManager, AuthenticationProvider, UserDetailsService.
   - Return user information (email, name and id)

This requires